nuitka/plugins/standard/PkgResourcesPlugin.py

```diff
@@ -48,11 +48,11 @@ def __init__(self):
             self.metadata = importlib_metadata
 
         try:
-            import metadata
-
-            self.metadata = metadata
+            from importlib import metadata
```

Comment:
This line is the actual fix.

Reply:
An, now I see what you are changing., OK.